config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'welcome#index'
+
+  get '/quotes', to: 'quotes#search'
```

Comment:
I can see why you might have named the action `search` since that is what we are doing. But if we think about where we are taken upon hitting that `search` button, we are being directed to an index page. So it might be better to keep it more RESTful and make an `index` action.